src/gql/queries/dataset.ts

```diff
@@ -59,6 +66,31 @@ export const GET_DATASET_SCHEMAS = gql`
     }
   }
 `;
+export const GET_DATASETS_LIST = gql`
+  query datasetsList($sceneId: ID!, $first: Int, $last: Int, $after: Cursor, $before: Cursor) {
```

Comment:
We now have three dataset queries. Can you investigate a little to see if you actually need to add this one?

Reply:
each one of queries take parameters and when I did pagination on them it didn't work so that why I though to add this new query